receiver/apachereceiver/scraper_test.go

```diff
@@ -56,7 +56,12 @@ func TestScraper(t *testing.T) {
 	expectedFile := filepath.Join("testdata", "scraper", "expected.json")
 	expectedMetrics, err := golden.ReadMetrics(expectedFile)
 	require.NoError(t, err)
+	url, err := url.Parse(apacheMock.URL)
+	require.NoError(t, err)
+	expectedMetrics.ResourceMetrics().At(0).Resource().Attributes().Remove("apache.server.port")
```

Comment:
Do we need to do this remove before the Put?

Reply:
Apparently we don't. I removed this line.